packages/public-api/src/middleware/common.ts

```diff
@@ -39,8 +39,7 @@ export function errorHandlerMiddleware(
     !(error instanceof UnauthorizedError) &&
     !(error instanceof ZodError)
   ) {
-    // TODO: add logger
-    console.error(error, req);
+    logger.error(error, req);
```

Comment:
Please make sure we log any 500 error without exposing a stack trace inside a response